### ISSUE TITLE
Got it to work once. Then it stopped working...

### ISSUE THREAD

**Issue by baterax** [ducfilan/Dark-mode-Franz-Ferdi#26]
And now nothing is dark anymore.
My eyes burn from all the ugly whiteness!
I have uninstalled Franz and deleted the folder under appdata/roaming MANY times.
Tried even copying the entire recipes frolder from your repository (instead of just the darkmode.css files), that broke it and made it so I couldn't add services.
Reinstalled, Franz is now working but still no darkness. Franz itself's dark mode works, and I can see the dark mode switch inside each service, but no luck getting them to work.
Also, is there one for instagram?
Thanks!

**Comment by ducfilan** [comment]
You mean all services?
Can you verify if `Setting` > `Join the Dark Side` is on and your service's "Enable Dark Mode" is on as well as the `darkmode.css` is placed in the right folder? (By taking screenshot)
Sometimes, you need to reload the service (by right click on the service's icon > select "Reload")

**Comment by baterax** [comment]
Hey!
Any light on this?
Or should I say, lack thereof. heheh

**Comment by ducfilan** [comment]
@baterax Looks like everything is set up correctly.
Can you reload the service few times to check if it applies?
---
If not, when you are in your service, can you select `View` > `Toggle Service Developer Tools` and see if any error occurs?

**Comment by baterax** [comment]
I reinstalled Franz from scratch a few times... And then re-added each service again. No luck.
The console does output a bunch of errors though:
![image](https://user-images.githubusercontent.com/8533902/84598264-6e12ad80-ae61-11ea-9033-da788a2d558c.png)

**Comment by baterax** [comment]
It looks the same for every service, except of course, for the path there.
Messenger, for instance:
![image](https://user-images.githubusercontent.com/8533902/84598291-9bf7f200-ae61-11ea-8083-84ff6c665652.png)

**Comment by ducfilan** [comment]
@baterax 
Can you remove all services and delete all folders in recipes folder and then add the services again?

**Comment by baterax** [comment]
Just did that. Still doesn't work...

**Comment by baterax** [comment]
Maybe just teamview into here and take a look? I'm sure it's something silly...

**Comment by baterax** [comment]
I just completely uninstalled Franz, reinstalled it, did it all over again and still no luck...

**Comment by ducfilan** [comment]
Maybe you placed in the wrong folder path.
Maybe it's `C:\Users\rodri\AppData...` instead of `C:\Users\Rod Duarte\AppData...`

**Comment by baterax** [comment]
I don't know why that screenshot shows "Rod Duarte" but that folder doesn't exist, I just checked that again:
![image](https://user-images.githubusercontent.com/8533902/84950214-77f91280-b0e6-11ea-98c3-b8f66b5bdbca.png)

**Comment by baterax** [comment]
If it was in the wrong folder, the "Dark mode" button wouldn't show up in the services configuration screen, right? And yet there it is:
![image](https://user-images.githubusercontent.com/8533902/84950723-3e74d700-b0e7-11ea-8b73-29c11150fb24.png)

**Comment by saulfg** [comment]
I have the same Issue, I have installed the 5.5.0 version of Franz and none of the service take the dark theme, I have the corresponding .css files into their corresponding receipes folder, also the dark themes on in each service and in franz, just a tought: can be this because the version of Franz?

**Comment by baterax** [comment]
Oh phew. Good to know it's not just me!

**Comment by baterax** [comment]
Any updates on this?

**Comment by baterax** [comment]
HELP

**Comment by baterax** [comment]
Just did a full fresh install of Windows.
Installed Franz and did it all from scratch.
Still doesn't work.
Please help...

**Comment by baterax** [comment]
Still at this! Hello? Please?

**Comment by ducfilan** [comment]
Sorry @baterax I cannot reproduce the problem.
If you use Mac, can you delete everything from `~/Library/Application Support/Franz/` and reinstall?

**Comment by baterax** [comment]
I'm on PC...
But as stated above, I've tried uninstalling everything and reinstalling several times.
The last time was a full reinstall of Windows, even!
Still no luck, though.

**Comment by ducfilan** [comment]
@baterax normal uninstalling won't delete all files.

**Comment by baterax** [comment]
I know.
I didn't just uninstall, I went in and deleted everything. Multiple times.
But also, like I said, I reset the entire PC, I formatted the hard drive, deleted the partition, reformatted and reinstalled Windows from scratch.
Then, when I installed Franz on this new, fresh Windows system, I had the problem again.

**Comment by ducfilan** [comment]
This sounds awkward. Sorry to say that I have no idea why @baterax

**Comment by ducfilan** [comment]
@baterax  Can you try with Google Calendar with this workaround method: https://github.com/ducfilan/Dark-mode-Franz-Ferdi/blob/fa46a1c64d8459cecac692825b79a7f04fb3fc81/googlecalendar/README.md

**Comment by baterax** [comment]
Alright.
Tried it, and it worked!
I don't even use Google Calendar with Franz, but, this worked. What now?

**Comment by baterax** [comment]
I'm guessing, since this worked, maybe you figured out how to make the others work again?

**Comment by ducfilan** [comment]
@baterax Can you go to this folder: `%appdata%/Franz/recipes/` and use my files for `messenger` and confirm if it works or not? https://drive.google.com/file/d/13-J0EJHUo6MRaRKMuSzp94M5_OYa4K2c/view?usp=sharing

**Comment by baterax** [comment]
YES!
IT DID WORK.
But, my folder isn't %appdata%/Franz/recipes/, it's %appdata%/roaming/Franz/recipes/.

Maybe that's the problem?

**Comment by ducfilan** [comment]
So it worked so you should put your files on the working folder. Is it ok?

**Comment by baterax** [comment]
No, I mean Messenger worked, the others still don't. I did place them in the right folders.

**Comment by baterax** [comment]
Or are you telling me to paste the Messenger files into the other folders? (Obviously, except the darkmode.css file)

**Comment by baterax** [comment]
Well this is interesting. Whatsapp now works all of a sudden. But Telegram still doesn't.

**Comment by baterax** [comment]
Yeah, I just need to get telegram to work now :-)

**Comment by baterax** [comment]
So it's been almost a year.
I still can't get Messenger and Instagram dark modes to work...
I've given up on Telegram.
WhatsApp is working but no luck with the other two.
I've reinstalled Franz, deleted and re-added the services, turned dark mode off, removed the darkmode.css files, restarted Franz and the PC multiple times, re-added darkmode files, re-enabled the toggle...
Nothing works. I'm sure it's something silly.
It's so weird that only WhatsApp works though, because I installed it exactly the same way as the other ones...
Please help?

**Comment by jampoyman** [comment]
Apologies for resurrecting an issue, @baterax can you check if the darkmode.css file that you downloaded had css code in it? I encountered the same issue, but upon checking I mistakenly downloaded the github html file and renamed it as darkmode.css.. This occured because I didnt use the raw link when I tried to download the file.
